HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/IEmailSender.cs

```diff
@@ -4,7 +4,15 @@ namespace HerPublicWebsite.BusinessLogic.ExternalServices.EmailSending;
 
 public interface IEmailSender
 {
-    public void SendReferenceCodeEmail
+    public void SendReferenceCodeLiveLAEmail
+    (
+        string emailAddress,
+        string recipientName,
+        string referenceCode,
+        string custodianCode
+    );
+
+    public void SendReferenceCodePendingLAEmail
```

Comment:
The method ordering is inconsistent between the interface and the implementation.

It doesn't much matter, but we should be consistent. Let's keep the "Live" one first (since it's the one for the ideal end state), so reorder the methods in the `GovUkNotifyApi` class.